docs/core/versions/index.md

```diff
@@ -7,130 +7,69 @@ ms.date: 02/13/2018
+.NET Core refers to the .NET Core Runtime and the .NET Core SDK which contains the tools you need to develop applications. A version of the .NET Core Runtime. .NET Core SDKs are designed to work with any previous version of the .NET Core Runtime.
 
-This article aims at clarifying how the .NET Core SDK and runtime are versioned.
+This article aims at clarifying version numbers for .NET Core SDK and .NET Core Runtime. [An explanation of version numbers for .NET Standard can be found here]().
 
-There are lots of moving parts that version independently in .NET Core. However, starting with .NET Core 2.0, there is an easy to understand top-level version number that everybody understands to be *the* version of ".NET Core" as a whole. The rest of this document goes into the details of the versioning of all those parts. These details can be important if you're a package manager, for example.
-
-> [!IMPORTANT]
-> The versioning details explained on this topic don't apply to the current version of the .NET Core SDK and runtime.
-> The version scheme is changing in future releases. You can see the current proposal at the [dotnet/designs](https://github.com/dotnet/designs/pull/29) repository.
+The central problem of versioning is that the .NET Core Runtime and .NET Core SDK add new features at a different rate - in general the .NET Core SDK provides updated tools more quickly than the .NET Core Runtime changes the runtime you use in production. Unfortunately, this problem has resulted in several versioning strategies over the last few years. You can learn about the [history of .NET Core versioning here]().
 
 ## Versioning details
 
-With .NET Core 2.0, downloads show a single version number in their file name. The following version numbers were unified:
-
-* The shared framework and associated runtime.
-* The .NET Core SDK and associated .NET Core CLI.
-* The `Microsoft.NETCore.App` metapackage.
-
-The use of a single version number makes it easier for users to know what version of the SDK to install on their dev machines, and what the corresponding version of the shared framework should be when time comes to provision a production environment. When downloading an SDK or runtime, the version number you see is going to be the same.
-
-### Installers
-
-With .NET Core 2.0, downloads for the [daily builds](https://github.com/dotnet/core-setup#daily-builds) and [releases](https://www.microsoft.com/net/download/core) adhere to a new naming scheme that is easier to understand.
-The installer UI in those downloads was also modified to clearly present the names and versions of the components being installed. In particular, titles now show the same version number that is in the download's file name.
+.NET Core 2.1 aligns to the .NET Core Runtime version number. .NET Core Runtime has a major/minor/patch approach to versioning that is roughly aligned with the [semantic versioning]([[Link to semantic versioning later in this doc]]).
```

Comment:
Instead of ".NET Core 2.1 aligns to the .NET Core Runtime version number.", consider "The major and minor version number identifies the .NET Core Runtime version number."